### PR TITLE
CI: Create Continuous Prerelease

### PR DESCRIPTION
**Pull request by HTRamsey**
This should upload commits to master to a continuous release making it easy for people to download daily builds

### CODE REVIEW

**Comment by HTRamsey** [comment]
@mrpollo Thoughts on this?

**Comment by mrpollo** [comment]
This should work, but we must test it as soon as possible. Unfortunately, we can't test until we merge.

By the way, I'm also using the same action package for PX4 here: https://github.com/PX4/PX4-Autopilot/pull/23703.

File: .github/actions/upload/action.yml [hunk 1]
```diff
@@ -43,3 +43,11 @@ runs:
       working-directory: ${{ runner.temp }}/shadow_build_dir/${{ inputs.source }}
       run: aws s3 cp ${{ inputs.artifact_name }} s3://qgroundcontrol/latest/${{ inputs.artifact_name }} --acl public-read
       shell: bash
+
+    - name: Create Continuous Release
+      if: ${{ github.event_name != 'pull_request' && github.ref_name == 'master' && !github.event.pull_request.head.repo.fork }}
```
Comment (mrpollo) [inline .github/actions/upload/action.yml:48]: Did you test `github.ref_name = 'master'`?  sometimes `ref_name` isn't what we want

Comment (mrpollo) [reply .github/actions/upload/action.yml:48]: I would skip the fork conditional, I don't its needed

Comment (mrpollo) [reply .github/actions/upload/action.yml:48]: Are you creating a release every time you push to master? Is this the desired behavior?

Comment (HTRamsey) [reply .github/actions/upload/action.yml:48]: Rather than every commit, should it be every night/week?

Comment (DonLakeFlyer) [reply .github/actions/upload/action.yml:48]: Daily builds upload to S3 on every merge so seems like it should match that.

**Review by DonLakeFlyer — APPROVED** [review]

File: .github/actions/upload/action.yml (hunk 1 above)
Comment (DonLakeFlyer) [reply .github/actions/upload/action.yml:48]: Also seems like this is way ore important on automating Stables. This just kinda duplicates the S3 thing which is referenced by the docs. I guess some people go here to get releases though.

**Pull request merged**